packages/storefront-lwc/src/modules/commerce/basket/basket.js

```diff
@@ -4,30 +4,38 @@
     SPDX-License-Identifier: BSD-3-Clause
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
-import { LightningElement } from 'lwc';
-import { ShoppingBasket } from 'commerce/data';
+import { LightningElement, wire } from 'lwc';
+import { GET_BASKET } from './gql.js';
+import { useQuery } from '@lwce/apollo-client';
 
 export default class Basket extends LightningElement {
-    products = [];
```

Comment:
dont need this anymore, logic simplified to just use the value from basket object.

Reply:
check line 33.